domains/_github-pages-challenge-gdsc-phenikaa.gdsc-phenikaa.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Caxtiq",
```

username doesnt match
```suggestion
        "username": "twilightty",
```